ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/views.py

```diff
@@ -42,6 +42,8 @@ def new_post(context, subsystem_id):
             }
 
     try:
+        organization = toolkit.get_action('organization_show')(context, {'id': form.get('organization')})
+        data_dict['organization_id'] = organization['id']
```

Comment:
Potential error: `organization` is fetched with an id recovered from `form` using `get`. This means the type of `organization` depends not only on the organization reference validity, but the existence of `organization` key in form. If the key must be in `form`, use `form['organization']` to fail at the correct location.